3.10/release-notes-new-features310.md

```diff
@@ -148,6 +148,57 @@ db._query(`FOR doc IN food_view
 See [Search highlighting with ArangoSearch](arangosearch-search-highlighting.html)
 for details.
 
+This feature is available in all editions but nested search queries are
+optimized in the Enterprise Edition only. The Community Edition does not utilize
+View indexes.
+
+### Nested search (Enterprise Edition)
+
+Nested search allows you to index arrays of objects in a way that lets you
+search the sub-objects with all the conditions met by a single sub-object
+instead of each condition being met by any of the sub-objects.
+
+Consider a document like the following:
+
+```json
+{
+  "dimensions": [
+    { "type": "height", "value": 35 },
+    { "type": "width", "value": 60 }
+  ]
+}
+```
+
+The following search query matches the document because the individual conditions
+are satisfied by one of the nested objects:
+
+```aql
+FOR doc IN viewName
+  SEARCH doc.dimensions.type == "height" AND doc.dimensions.value > 40
+  RETURN doc
+```
+
+If you only want to find documents where both conditions are true for a single
+nested object, you could post-filter the search results:
+
+```aql
+FOR doc IN viewName
+  SEARCH doc.dimensions.type == "height" AND doc.dimensions.value > 40
+  FILTER LENGTH(doc.dimensions[* FILTER CURRENT.type == "height" AND CURRENT.value > 40]) > 0
+  RETURN doc
+```
+
+The new nested search feature allows you to simplify the query and get better
+performance:
+
+```aql
+FOR doc IN viewName
+  SEARCH doc.dimensions[? FILTER CURRENT.type == "height" AND CURRENT.value > 40]
+  RETURN doc
+```
+
+See [Nested search in ArangoSearch](arangosearch-nested-search.html) for details.
+
 This feature is only available in the Enterprise Edition.
```

Comment:
The above text says it is available in all editions (not just enterprise), but optimized only in enterprise.